.github/workflows/build-rusEFI-console.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
       - name: Install Tools
         run: sudo apt-get install ncftp
 
+      - name: Generate Java (Antlr)
+        working-directory: ./java_tools/configuration_definition
```

Comment:
is "./" providing any value here?

Reply:
Yes.  `./` forces it to look in the current directory, instead of looking in `$PATH`.

Reply:
TIL thank you